webgoat-container/src/main/java/org/owasp/webgoat/assignments/AssignmentEndpoint.java

```diff
@@ -41,7 +41,8 @@
  * </p>
  * Note: each subclass should declare this annotation otherwise the WebGoat framework cannot find your endpoint.
  */
-public abstract class AssignmentEndpoint extends Endpoint {
+//TODO: rename to assignment
```

Comment:
Do you still want to rename this in this pull request or in a feature change?

Reply:
And would it not be strange to have two Assignment classes in that case

Reply:
Correct, will remove this